request.ts

```diff
@@ -1,4 +1,4 @@
-const githubApiUrl = "https://api.github.com";
+const githubApiUrl = Deno.env.get("GITHUB_API_URL") ?? "https://api.github.com";
```

Comment:
This will break existing scripts (if they don't pin the current release version) because doing `Deno.env.get` requires more permissions. So if this is installed without `--allow-env`, this line will throw an exception.

I'll try to wrap it into a check whether the script has permissions to access env, so that if it doesn't we can still use the default URL.

Reply:
Great point, I was just using `-A` and hadn't thought about that. Very naughty of me, I'll update my testing procedure.